README.md

```diff
@@ -60,14 +60,30 @@ release a new version, update the version number in `version.rb`, and then run
 git commits and tags, and push the `.gem` file to
 [rubygems.org](https://rubygems.org).
 
+## Documentation
+
+We maintain API documentation using [YARD](https://yardoc.org/), which is
+published automatically at
+[RubyDoc.info](https://rubydoc.info/github/panorama-ed/memo_wise/main).
+
+To edit documentation locally and see how it will be rendered, try running:
```

Comment:
Nitpick: `try running` to me sounds like we're not confident these steps will work. Can we remove `try`?

Reply:
Yes! Thanks